footprints/main/tests/test_views.py

```diff
@@ -462,62 +457,72 @@ def test_export_list(self):
-                            self.footprint1.actors()))
-
-        row1 += ','.join(interpolate_role_actors(
-                            Role.objects.all().for_imprint(),
-                            self.footprint1.book_copy.imprint.actor.all()))
-
-        row1 += '\r\n'
-
-        row2 = ('Odyssey,'  # Footprint Title
-                'c. 1984,'  # Footprint Date
-                '"Cracow, Poland",'  # Footprint Location
-                '{},'  # Footprint Owners
-                '{},'  # Written Work Title
-                '"The Odyssey, Edition 1",'  # Imprint Display Title
-                '{},'  # Imprint Printers
-                'c. 1984,'  # Imprint Creation Date
-                '{},'  # Footprint Creation Date
-                '90,'  # Footprint Percent Complete
-                '{},'  # Literary Work LOC
-                '{},'  # Imprint Actor and Role
-                ','  # Imprint BHB
-                ','  # Imprint OCLC Number
-                'Medium,'  # Evidence Type
-                'Provenance,'  # Evidence Location
-                'call number,'  # Evidence Call Number
-                'lorem ipsum,'  # Evidence Details
-                ).format(
-            o, self.footprint2.book_copy.imprint.work.title,
-            p, self.footprint2.created_at.strftime('%m/%d/%Y'),
-            self.footprint2.book_copy.imprint.work
-                    .get_library_of_congress_identifier(), actors)
+        row1 = ['Empty Footprint', 'None', 'None', '', 'None', 'None', '',
+                'None', self.footprint1.created_at.strftime('%m/%d/%Y'),
+                0, 'None', '', '', '', '', '', 'None', 'None']
+
+        row1 += interpolate_role_actors(Role.objects.all().for_footprint(),
+                                        self.footprint1.actors())
+
+        row1 += interpolate_role_actors(
+            Role.objects.all().for_imprint(),
+            self.footprint1.book_copy.imprint.actor.all())
+
+        work = self.footprint2.book_copy.imprint.work
+        row2 = ['Odyssey',  # Footprint Title
+                'c. 1984',  # Footprint Date
+                'Cracow, Poland',  # Footprint Location
+                o,  # Footprint Owners
+                work.title,  # Written Work Title
+                'The Odyssey, Edition 1',  # Imprint Display Title
+                p,  # Imprint Printers
+                'c. 1984',  # Imprint Creation Date
+                self.footprint2.created_at.strftime('%m/%d/%Y'),
+                90,  # Footprint Percent Complete
+                'None',
+                actors,  # Imprint Actor and Role
+                '',  # Imprint BHB
+                '',  # Imprint OCLC Number
+                'Medium',  # Evidence Type
+                'Provenance',  # Evidence Location
+                'call number',  # Evidence Call Number
+                'lorem ipsum']
 
         # Footprint Actors
-        row2 += ','.join(interpolate_role_actors(
```

Comment:
I modified the test to go directly against `get_rows` to work around SOLR test limitations. `get_rows` returns an array, so I needed to modify what was being compared.